lib/internal/util/inspect_colors.js

```diff
@@ -0,0 +1,505 @@
+'use strict';
+
+const {
+  ArrayIsArray,
+  ArrayPrototypeIncludes,
+  ArrayPrototypePush,
+  MathAbs,
+  MathMax,
+  MathMin,
+  MathRound,
+  NumberParseInt,
+  RegExpPrototypeExec,
+  SafeMap,
+  StringPrototypeSplit,
+  StringPrototypeTrim,
+} = primordials;
+
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+// https://github.com/denoland/deno/blob/ece2a3de5b19588160634452638aa656218853c5/ext/console/01_console.js#L2575
+const colorKeywords = new SafeMap([
```

Comment:
Minor nit... can populating this map be done lazily? Just concerned that this is a fairly large number of strings to create when they are only used in certain situations.